templates/Makefile

```diff
@@ -30,20 +30,23 @@ imagedev:
 
 vendor: imagedev
 	$(runcmd) ./hack/vendor.sh
-	chown -R $(USER):$(USER) ./vendor
-	chown -R $(USER):$(USER) ./Godeps
+	sudo chown -R $(USER):$(USER) ./vendor
+	sudo chown -R $(USER):$(USER) ./Godeps
```

Comment:
Could you clarify?

Reply:
files created inside the container on a volume mounted from the host will be owned by root when the container exits =/, would love some alternative from this crap

Reply:
We went through this last week. Funny thing that docker for mac preserves the host owership.

<img width="1128" alt="screen shot 2017-10-14 at 10 59 51" src="https://user-images.githubusercontent.com/15719/31576330-00d087ac-b0cf-11e7-811e-4569f496bf2a.png">

Reply:
Probably because docker on Mac OS is actually running inside a VM not directly on the host using namespaces to create isolation (like what is done on linux). Since a VM can be running even another OS it makes sense that data written there in the end will have the ownership of the user that is running the VM. But to be honest, I known very little on how this kind of thing works on virtualization/hypervisor stuff.